specs/core/0_beacon-chain.md

```diff
@@ -185,6 +185,7 @@ These configurations are updated for releases, but may be out of sync during `de
 | `MAX_ATTESTATION_PARTICIPANTS` | `2**12` (= 4,096) |
 | `MIN_PER_EPOCH_CHURN_LIMIT` | `2**2` (= 4) |
 | `CHURN_LIMIT_QUOTIENT` | `2**16` (= 65,536) |
+| `BASE_REWARDS_PER_EPOCH` | `5` |
```

Comment:
I obviously like this :) I was wondering whether this is actually an exact figure though: I am confused by PROPOSER_REWARD_QUOTIENT, which seems to make this 1/8 in one of the cases.

Reply:
There's actually only four base rewards now that the inclusion delay bonus is an inclusion delay penalty:

1) Matching FFG source
2) Matching FFG target
3) Matching head
4) Matching crosslink data root

Yay for powers of two 🤪

Reply:
So should this be changed to 4 then?

How about block proposer rewards?